.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,13 @@
+# Approve PRs with "pr/auto-approve". mergify takes care of the actual merge.
+
+name: auto-approve
+on: pull_request
+
+jobs:
+  auto-approve:
+    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2.0.0
+      with:
+        github-token: "${{ secrets.AUTO_APPROVE_GITHUB_TOKEN }}"
```

Comment:
We already have such a workflow, which does not use `pr/auto-approve`, and instead triggers on the author...

https://github.com/aws/jsii/blob/716a8994dc46cc44d38ae61289a2193b3b16ebec/.github/workflows/dependabot.yml#L1-L28

Why create yet another one?

Reply:
The `pr/auto-approve` is more generic and can be applied to additional automation tasks. This is what we are normalizing on for the CDK repo.

Reply:
Deleted the old one

Reply:
Well sure but you are giving away one property of the old workflow with __YOU__ had insisted be added: the ability to only auto-approve changes that have an impact of "minor" or less (i.e: the dependency is verified not to be a major version bump).

Reply:
Not sure I understand. Isn't this taken care of by the yarn-update workflow?

Reply:
okay, got it. this workflow only auto-approve dependabot PRs for minor versions and not major versions.